theme/management/commands/dump_user_quota.py

```diff
@@ -99,7 +102,13 @@ def handle(self, *args, **options):
                 uz
             ]
             if expand:
-                quota_resources = user.uaccess.owned_resources.filter(quota_holder=user)
+                try:
+                    quota_resources = user.uaccess.owned_resources.filter(quota_holder=user)
+                except (ValidationError, PermissionDenied) as e:
```

Comment:
Why would there be `ValidationError `when querying the DB?

Reply:
good call